pkg/utils/kubernetes/client/mock/gomock_reflect_888247816/prog.go

```diff
@@ -0,0 +1,66 @@
+package main
```

Comment:
Remove this file and also `pkg/utils/kubernetes/client/mock/gomock_reflect_888247816/prog.bin`